content/learning-paths/cross-platform/kleidiai-explainer/page2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: KleidiAI in a real software stack
+weight: 3
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## High-level KleidiAI architecture
+This section aims to provide an abstracted overview of KleidiAI's components before diving into the specifics. The KleidiAI source files are publicly accessible in the [KleidiAI GitLab repository](https://gitlab.arm.com/kleidi/kleidiai). Navigate there in your web browser to follow along and understand KleidiAI's structure.
+
+KlediAI's micro-kernels are located in the `/kai/ukernels/matmul` directory; navigate there now. There are essentially two types of KleidiAI micro-kernels:
+1. Quantizing/Packing routines    - under the `pack` directory.
+2. Matrix Multiplication routines - the three directories starting with `matmul_clamp`. Each directory contains routines specialized for a specific input data type.
+
+
+![KleidiAI stuff](KleidiAI-src.jpg "Figure 3. KleidiAI src directory")
```

Comment:
This is broken, I think the extension should be capitalized .JPG

Reply:
updated to capital